gaudimarlinwrapper looking for cmake file not in Gaudi 35.0

This seems to be because the files are different with Gaudi 35.0

CMake Warning at CMakeLists.txt:50 (find_package):
  By not providing "FindGaudiProject.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "GaudiProject", but CMake did not find one.
  Could not find a package configuration file provided by "GaudiProject" with
  any of the following names:
    GaudiProjectConfig.cmake
    gaudiproject-config.cmake
  Add the installation prefix of "GaudiProject" to CMAKE_PREFIX_PATH or set
  "GaudiProject_DIR" to a directory containing one of the above files.  If
  "GaudiProject" provides a separate development package or SDK, be sure it
  has been installed.
CMake Error at CMakeLists.txt:55 (gaudi_project):
  Unknown CMake command "gaudi_project".

--- a/config/packages.yaml
+++ b/config/packages.yaml
@@ -29,6 +29,8 @@ packages:
   # Avoid concretizer conflict for edm4hep
   pythia8:
     version: [8303]
+  gaudi:
+    version: [34.0]
   # gaudi v34 does not work with python 3.8
   # setting the version explicitly avoids concretizer errors
   python:

